Framework/PythonInterface/mantid/geometry/src/Exports/Instrument.cpp

```diff
@@ -46,11 +46,13 @@ void export_Instrument() {
            "represents the source")
 
       .def("getComponentByName",
+           // cppcheck-suppress cstyleCast
```

Comment:
cstylecast used in all such exports, suspect it is not worth changing them all under the scope of this hackathon. Will raise an issue.